vnext/ReactWindowsCore/ReactWindowsCore.vcxproj

```diff
@@ -91,9 +91,6 @@
         $(FollyDir);
         $(ReactNativeWindowsDir)\ReactWindowsCore\pch;
         $(ReactNativeWindowsDir)\ReactWindowsCore\tracing;
-        $(ProjectDir);
```

Comment:
Not directly related to the PR, but these shouldn't be here. ProjectDir will already be included, and the latter two allow consuming from build output while building which is dangerous and not really that useful.